.github/workflows/cypress-test.yml

```diff
@@ -16,21 +16,28 @@ env:
 jobs:
   tests:
     name: Run Cypress tests
-    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest , windows-latest ]
+        jdk: [ 11, 17 ]
```

Comment:
Yes we do not need jdk set up for this repo's action, however, me and @scrawfor99 found out that we do need this step for windows runner for the `run opensearch with single plugin` action in security repo. Otherwise it will give an error of `java.lang.UnsupportedClassVersionError`.
<img width="1301" alt="Screenshot 2023-01-31 at 9 03 40 PM" src="https://user-images.githubusercontent.com/109499885/215956195-56d1f69e-b968-4acb-bc33-7b65cd7eae32.png">

Reply:
Lets just use one of the JDKs, I'd go with 11

Reply:
Done

Reply:
Done